changelogs/1.22.9.yaml

```diff
@@ -0,0 +1,6 @@
+date: March 22, 2023
```

Comment:
for clarity - this is pulling in the changelog from the previous version - this branches changelog is updated below